apps/nextjs-website/next.config.js

```diff
@@ -8,7 +8,7 @@
  */
 const nextConfig = {
   reactStrictMode: true,
-  transpilePackages: ['@pagopa/mui-italia'],
+  // transpilePackages: ['@pagopa/mui-italia'],
```

Comment:
To be able to use `mui-italia` theme we need this line.

```suggestion
  transpilePackages: ['@pagopa/mui-italia'],
```